### ISSUE TITLE
"Response Class" => "Model": multiple tooltips when hovered

### ISSUE THREAD

**Issue by agongdai** [swagger-api/swagger-ui#1836]
After I upgraded SwaggerUI from version 2.1.3 to 2.1.4 (to include the JSON editor), there seems a bug. Hovering to one property seems triggers all tooltips. This issue doesn't exist in version 2.1.3.

![2015-12-25_150858](https://cloud.githubusercontent.com/assets/1829273/12000944/35209a68-ab1a-11e5-929d-e478df15bea2.png)

**Comment by NicolasCayet** [comment]
I've seen the same problem in master branch. Weird thing: the issue is not present in the 2.1.8 tag....

**Comment by xiehan** [comment]
Thank goodness someone else ran into this; I spent days trying to figure out why this was happening on my local fork but not on [petstore.swagger.io](http://petstore.swagger.io) or [online.swagger.io](http://online.swagger.io) (more on that in a minute).

Anyway, the issue is in swagger-js -- I think it's a matter of a `<div>` not being closed properly somewhere, but I'm not super familiar with that code so I'm not positive. Anyway, the clue was that the markup for the signatures is being rendered totally wrong; the trigger for the tooltips is a `<span class="propWrap">`, but when I looked up at my markup, the entire contents of my model were being wrapped in `<span class="propWrap">`, and any subsequent child models were being rendered inside of there as well. It's a mess.

I've opened a ticket there -- swagger-api/swagger-js#674

Now regarding the [petstore.swagger.io](http://petstore.swagger.io) / [online.swagger.io](http://online.swagger.io) conundrum -- what was really driving me crazy was not being able to figure out why those sites were rendering the tooltips properly, but my own fork of Swagger-UI wasn't, even though (as far as I can tell) the swagger.io sites are serving the latest code, v2.1.4. But then I realized that the source code for swagger-js (swagger-client.js) was not up-to-date with the latest official build of swagger-js; it's still using v2.1.6. I'm not sure when/how those sites get updated, but if there's a way to either have them continuously deployed, or at least put a timestamp in the header of the swagger-ui.js file so we know how recently it's been updated, that'd be swell.
